source/fuzz/fuzzer_context.cpp

```diff
@@ -152,11 +152,12 @@ const std::pair<uint32_t, uint32_t> kChanceOfReplacingParametersWithGlobals = {
     30, 70};
 const std::pair<uint32_t, uint32_t> kChanceOfReplacingParametersWithStruct = {
     20, 40};
+const std::pair<uint32_t, uint32_t>
+        kChanceOfSwappingAnotherPairOfFunctionVariables = {30, 90};
```

Comment:
We should move this so that these constants appear in alphabetical order. (Perhaps it wasn't like that already, but it should be.)

So: before kChanceOfSwappingConditionalBranchOperands, but after kChanceOfSplittingBlock.

Could you do that please?

Reply:
Also the formatting check fails on this file, so please ensure you fix that.

Locally you can run:

./utils/check_code_format.sh

as long as clang-format version 5 is on your path.

Reply:
> We should move this so that these constants appear in alphabetical order. (Perhaps it wasn't like that already, but it should be.)
> 
> So: before kChanceOfSwappingConditionalBranchOperands, but after kChanceOfSplittingBlock.
> 
> Could you do that please?

I seemed like I compared "Another" with "Block" but actually 
"splitting" goes before "swapping". Fixed.

Reply:
> Also the formatting check fails on this file, so please ensure you fix that.
> 
> Locally you can run:
> 
> ./utils/check_code_format.sh
> 
> as long as clang-format version 5 is on your path.

I just clang formatted `fuzzer_context.cpp`. It should pass the test now.